srcs/incs/HttpDeleteResponse.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef SRCS_INCS_HTTPDELETERESPONSE_HPP_
+#define SRCS_INCS_HTTPDELETERESPONSE_HPP_
+
+#include <string>
+#include "IResponse.hpp"
+
+class HttpDeleteResponse: public IResponse {
```

Comment:
In the graph you made, the concrete HttpResponses are inheriting from the AHttpResponse class, but then they are returning a pointer to the IResponse interface... don't you think the relationship between interfaces and its concrete classes is the same that exists between abstract classes and their concrete implementation?
You are actually inheriting from IResponse, but then... what is AHttpResponse? Is the graph wrong? What is your Inheritance tree in the end?
I don't get it.

Reply:
The `IResponse` interface is the contract between the `Connection` class and the concrete response. The `AhttpResponse` is just a class that holds related method to all concrete responses. This was a first approach of gathering all related behaviour when making the scaffolding but will evolve for sure!